storage/io/src/main/java/org/openscience/cdk/io/ShelXReader.java

```diff
@@ -256,7 +256,6 @@ private ICrystal readCrystal(ICrystal crystal) throws IOException {
             } else if (command.equalsIgnoreCase("FRAG")) {
             } else if (command.equalsIgnoreCase("FEND")) {
             } else if (command.equalsIgnoreCase("EXYZ")) {
-            } else if (command.equalsIgnoreCase("EXTI")) {
```

Comment:
Okay, was that really needed? :)

BTW, this was actually part of my PhD thesis... I really hope to find some time to go back to this code and make my PhD work more accessible from the CDK...

Reply:
TBF I checked the manual and it really can't occur there :-).

Reply:
Another bug.